Same thing, new syntax

rolandomunoz opened the pull request:
Commit Summary:
	- Changed: field names from the dialogue box. (Now it is similar to Create Corpus...)
	- Changed: old code to new syntax

File changes:
	modified:   get_durations.praat

Commit:
Commit Summary:
	- Changed: field names from the dialogue box. (Now it is similar to Create Corpus...)
	- Changed: old code to new syntax

File changes:
	modified:   get_durations.praat

Commit:
Changed variable files to sound_file_extension

jeanphilippegoldman commented on get_durations.praat:
```diff
@@ -1,36 +1,37 @@
-
+# get durations of audio files in a folder
+# author: jeanphilippegoldman@gmail.com
+# Description: ?
```
put line 1

rolandomunoz replied:
Done. Now:

```
# author: jeanphilippegoldman@gmail.com
# Description: get durations of audio files in a folder
```
is this right?

jeanphilippegoldman commented on get_durations.praat:
```diff
@@ -1,36 +1,37 @@
+writeInfoLine: "Get durations..."
+
+for ifile to numberOfFiles
+  sd$ = object$[fileList, ifile]
```
explain sd

jeanphilippegoldman commented on get_durations.praat:
```diff
@@ -1,36 +1,37 @@
+folder$= folder$-"\"-"/"
+runScript: "setbasename.praat", folder$, ""
+
+fileList = Create Strings as file list: "list", folder$ + "/*" + sound_file_extension$
```
let keep variable names with underscore convention instead of camelCase (to distinguish from procedures), right ?

rolandomunoz replied:
Done!

jeanphilippegoldman reviewed (changes requested):
please review my 2 comments

Commit:
- Changed camelCase convention to underscore
- Added description in the header

Commit:
- Changed syntax code
- Changed file organization
- Added recursive search procedure
- Added recursive search function to open_files3.praat
